### PR TITLE
Add perimeter in Measurement addons

### PR DESCRIPTION
**Pull request by pierrejego**
This pull request add perimeter measurement in addons

It also add metric (meter and kilometer) to have more details.

### CODE REVIEW

**Comment by landryb** [comment]
Minor stylistic nit: configure your editor/IDE to avoid removing the newline at the end-of-file. POSIX said a file should end up by a newline char, but windows doesnt care about POSIX.. also fix the whitespace changes adding spaces to an empty line when it's not needed (ie Annotation.js line 404)

The code duplication in `showPopup()` in the `if (this.measurePerimeterControl.active)` block can be simplified, the only difference i see is the `tpl:` value...

Good catch on adding the missing `addons/measure/config.json` !
And congrats for your first PRs :)

File: mapfishapp/src/main/webapp/app/addons/measure/js/main.js [hunk 1]
```diff
@@ -92,43 +105,84 @@ GEOR.Addons.Measure = Ext.extend(GEOR.Addons.Base, {
...
+        var verticesGeom = [];
+        if (points.length != 0){
+        	for(i=0; i < points.length; i++){
+        		var point = new OpenLayers.Geometry.Point(points[i].x, points[i].y);
+        		verticesGeom.push(point);
+        	};        	
+        	var ring = new OpenLayers.Geometry.LinearRing(verticesGeom);
+            distGeom = Math.round(ring.getGeodesicLength(this.map.getProjectionObject()));
+            distGeomUnits = "m";
+        	console.log(distGeom + ' ' + distGeomUnits);
```
Comment (fvanderbiest) [inline mapfishapp/src/main/webapp/app/addons/measure/js/main.js]: no console.log please

File: mapfishapp/src/main/webapp/app/addons/annotation/js/FeaturePanel.js [hunk 2]
```diff
@@ -267,6 +267,37 @@ GEOR.FeaturePanel = Ext.extend(Ext.form.FormPanel, {
...
         }
 
+        // Add item within feature panel for polygon, rectangle and ring  
+        if (feature.geometry.CLASS_NAME === "OpenLayers.Geometry.Polygon" ) {
+            if (!feature.isLabel) {
+                // Opacity
+                oItems.push({
+                	id:'opacitySlider',
+                	xtype: 'sliderfield',
+                    name: 'Opacity',
```
Comment (fvanderbiest) [inline mapfishapp/src/main/webapp/app/addons/annotation/js/FeaturePanel.js]: indents

**Comment by fvanderbiest** [comment]
Since this feature is already done in the new measurements addon, I'm closing this one.

**Pull request closed without merging**